src/.vsconfig

```diff
@@ -0,0 +1,94 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.VisualStudio.Component.CoreEditor",
+    "Microsoft.VisualStudio.Workload.CoreEditor",
+    "Microsoft.NetCore.Component.Runtime.3.1",
+    "Microsoft.NetCore.Component.SDK",
+    "Microsoft.VisualStudio.Component.NuGet",
+    "Microsoft.VisualStudio.Component.Roslyn.Compiler",
+    "Microsoft.VisualStudio.Component.Roslyn.LanguageServices",
+    "Microsoft.NetCore.Component.DevelopmentTools",
+    "Microsoft.VisualStudio.ComponentGroup.WebToolsExtensions",
+    "Microsoft.VisualStudio.Component.DockerTools",
+    "Microsoft.NetCore.Component.Web",
+    "Microsoft.Net.ComponentGroup.DevelopmentPrerequisites",
+    "Microsoft.VisualStudio.Component.TypeScript.4.0",
+    "Microsoft.VisualStudio.Component.JavaScript.TypeScript",
+    "Microsoft.VisualStudio.Component.JavaScript.Diagnostics",
+    "Microsoft.Component.MSBuild",
+    "Microsoft.VisualStudio.Component.TextTemplating",
+    "Component.Microsoft.VisualStudio.RazorExtension",
+    "Microsoft.VisualStudio.Component.IISExpress",
+    "Microsoft.VisualStudio.Component.SQL.ADAL",
+    "Microsoft.VisualStudio.Component.SQL.LocalDB.Runtime",
+    "Microsoft.VisualStudio.Component.Common.Azure.Tools",
+    "Microsoft.VisualStudio.Component.SQL.CLR",
+    "Microsoft.VisualStudio.Component.MSODBC.SQL",
+    "Microsoft.VisualStudio.Component.MSSQL.CMDLnUtils",
+    "Microsoft.VisualStudio.Component.ManagedDesktop.Core",
+    "Microsoft.VisualStudio.Component.SQL.SSDT",
+    "Microsoft.VisualStudio.Component.SQL.DataSources",
+    "Component.Microsoft.Web.LibraryManager",
+    "Microsoft.VisualStudio.ComponentGroup.Web",
+    "Microsoft.VisualStudio.Component.Web",
+    "Microsoft.VisualStudio.Component.IntelliCode",
+    "Component.Microsoft.VisualStudio.LiveShare",
+    "Microsoft.VisualStudio.ComponentGroup.Web.Client",
+    "Microsoft.Net.ComponentGroup.TargetingPacks.Common",
+    "Component.Microsoft.VisualStudio.Web.AzureFunctions",
```

Comment:
Can we confirm that all these components are required - not sure somethign like AzureFunctions should be required

Reply:
Indeed, didn't saw that file